htlcswitch/switch_test.go

```diff
@@ -1830,10 +1830,6 @@ func TestSwitchSendPayment(t *testing.T) {
 		t.Fatal("request was not propagated to destination")
 	}
 
-	if s.numPendingPayments() != 1 {
```

Comment:
Why remove rather than modify to check the new persistent store?

Reply:
No concept of pending payments anymore.